cypress/e2e/firstTest.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('First Test', () => {
+
+    it('Check the page', () => {
+        cy.visit('http://localhost:4200/pages/layout/stepper');
+        cy.contains("Step content #1");
+        cy.contains("next").click();
+        cy.contains("Step content #2");
+        cy.contains("next").click();
+        cy.contains("Step content #3");
+        cy.contains("next").click();
+        cy.contains("Step content #4");
```

Comment:
Small improvement here
```suggestion
    cy.get('.horizontal > .step-content > h3.ng-star-inserted').as("step")
    cy.get("@step").should("contain", "Step content #1");
    cy.contains("next").click();
    cy.get("@step").should("contain", "Step content #2");
    cy.contains("next").click();
    cy.get("@step").should("contain", "Step content #3");
    cy.contains("next").click();
    cy.get("@step").should("contain", "Step content #4");
```